### PR TITLE
Get rank info

### PR DESCRIPTION
**Pull request by yurireeis**
A simple request that brings the user week or month leaderboard.

### CODE REVIEW

File: duolingo.py [hunk 1]
```diff
@@ -1,9 +1,12 @@
 """Unofficial API for duolingo.com"""
+import pdb
```
Comment (darkSasori) [inline duolingo.py]: Remove this line.

File: README.md [hunk 2]
```diff
@@ -589,3 +589,19 @@ get_vocabulary.
...
 ]
 ```
+
+#### get_leaderboard(unit=None, before=time.time())
+
+Returns an ordered list containing logged user leaderboard. 
+You need to bring week or month as a unit to get the desired result.
+The before argument come with time.time() function, but if you need to know what's 
+your leaderboard in another date, you can pass the date in a epoch format 
+(str works fine too!)
```
Comment (darkSasori) [inline README.md]: Dont be ambiguous, this parameter receive float.

File: duolingo.py (hunk 1 above)
Comment (yurireeis) [inline duolingo.py]: Done

File: README.md (hunk 2 above)
Comment (yurireeis) [inline README.md]: Done

File: README.md [hunk 3]
```diff
@@ -589,3 +589,18 @@ get_vocabulary.
...
+
+Returns an ordered list containing logged user leaderboard. 
+You need to bring week or month as a unit to get the desired result.
+The before argument come with time.time() function, but if you need to know what's 
+your leaderboard in another date, you can pass the date in a epoch format
+
+```py
+>>> lingo = duolingo.Duolingo('kartik')
+>>> print lingo.get_leaderboard('week')
+'https://www.duolingo.com/friendships/leaderboard_activity?unit=week&_=1484054970'
```
Comment (KartikTalwar) [inline README.md]: can this include a sample dictionary output the function returns instead of the url

File: duolingo.py [hunk 4]
```diff
@@ -75,6 +75,37 @@ def get_activity_stream(self, before=None):
         except:
             raise Exception('Could not get activity stream')
 
+    def get_leaderboard(self, unit=None, before=time.time()):
```
Comment (KartikTalwar) [inline duolingo.py]: `time.time()` returns a float, probably want int?

Comment (yurireeis) [inline duolingo.py]: maybe int, float or str... the argument before in this case will be parsed in str in the line 91 as follow:

`url = url.format(unit, before)`

Anyway, to follow the pattern that is implemented in previous methods, I'll set before as a str in datetime format:

`    def get_leaderboard(self, unit=None, before=None):
        """
        Get user's rank in the week in descending order, stream from
        ``https://www.duolingo.com/friendships/leaderboard_activity?unit=week&_=time

        :param before: Datetime in format '2015-07-06 05:42:24'
        :param unit: maybe week or month
        :type before: str
        :type unit: str
        :rtype: List
        """
        if unit:
            url = 'https://www.duolingo.com/friendships/leaderboard_activity?unit={}&_={}'
        else:
            raise Exception('Needs unit as argument (week or month)')

        if before:
            url = url.format(unit, before)
        else:
            raise Exception('Needs str in Datetime format "%Y.%m.%d %H:%M:%S"')`

Comment (KartikTalwar) [inline duolingo.py]: 👍

**Pull request merged**